development/compiling/compiling_with_mono.rst

```diff
@@ -77,8 +77,8 @@ without having to repeat this process.
 ``<godot_binary>`` refers to the tools binary you compiled above with the Mono
 module enabled. Its exact name will differ based on your system and
 configuration, but should be of the form
-``bin/godot.<platform>.tools.<bits>.mono``, e.g. ``bin/godot.linuxbsd.tools.64.mono``
-or ``bin/godot.windows.tools.64.mono.exe``. Be especially aware of the **.mono**
+``bin/godot.<platform>.editor.<bits>.mono``, e.g. ``bin/godot.linuxbsd.editor.64.mono``
```

Comment:
File names use `arch` instead `bits` as well.

Reply:
on linux the bits field is replaced with `x86_64`, is that the same on other platforms (like windows)?

```suggestion
``bin/godot.<platform>.editor.<arch>.mono``, e.g. ``bin/godot.linuxbsd.editor.x86_64.mono``
```

Reply:
Yes.